fontc/src/lib.rs

```diff
@@ -2432,4 +2432,27 @@ mod tests {
     fn compile_obeys_no_export_designspace() {
         assert_noexport("designspace_from_glyphs/WghtVar_NoExport.designspace");
     }
+
+    fn assert_fs_type(source: &str, expected_fs_type: u16) {
+        let temp_dir = tempdir().unwrap();
+        let build_dir = temp_dir.path();
+        compile(Args::for_test(build_dir, source));
+
+        let font_file = build_dir.join("font.ttf");
+        let buf = fs::read(font_file).unwrap();
+        let font = FontRef::new(&buf).unwrap();
+        let os2 = font.os2().unwrap();
+
+        assert_eq!(expected_fs_type, os2.fs_type());
```

Comment:
one thing to be aware of with this pattern of having the assert outside of the test case is that, if you were calling this multiple times from the same test it can end up being slightly annoying to figure out which of the calls is failing. In that case I find it's nicer to have your helper function just return a bool or something, and then you can have the `assert!` be in the test case instead.


This doesn't matter here, since it's only called once per test, but just wanted to point it out as a general tip.

Reply:
I agree that if we called it many times from the same test it would need more context or a return. That would be a very odd usage here.